test/helpers/fusionUtils.js

```diff
@@ -24,23 +38,27 @@ async function buildCalldataForOrder({
         auction: { startTime: auctionStartTime, details: auctionDetails },
     } = setupData;
 
-    let postInteractionFeeDataTypes = ['uint8'];
-    let postInteractionFeeData = [0];
+    let postInteractionFeeDataTypes = [];
+    let postInteractionFeeData = [];
     if (feeType === 1) {
-        postInteractionFeeDataTypes = [...postInteractionFeeDataTypes, 'bytes4'];
-        postInteractionFeeData = [feeType, '0x' + resolverFee.toString(16).padStart(8, '0')];
+        postInteractionFeeDataTypes = ['bytes4'];
+        postInteractionFeeData = ['0x' + resolverFee.toString(16).padStart(8, '0')];
     }
     if (feeType === 2) {
-        postInteractionFeeDataTypes = [...postInteractionFeeDataTypes, 'bytes20', 'bytes4'];
-        postInteractionFeeData = [feeType, integrator, '0x' + resolverFee.toString(16).padStart(8, '0')];
+        postInteractionFeeDataTypes = ['bytes20', 'bytes4'];
+        postInteractionFeeData = [integrator, '0x' + resolverFee.toString(16).padStart(8, '0')];
+    }
+    if (feeType > 2) {
```

Comment:
It seems that we can't test `feeType === 3` (both integrator and resolver fees) here, I would add this feature and test just in case.

Reply:
I reverted these changes because currently, we lack tests that cover scenarios with both fees simultaneously, and the helper method in question constructs extraData for only one of them.
I will create a task to add tests covering scenarios with both fees to ensure our codebase is thoroughly tested in these situations